example/lib_ops/gemm_lib.cc

```diff
@@ -123,13 +114,17 @@ MXReturnValue backward(std::map<std::string, std::string> attrs,
 
   std::cout << "n: " << n << " k: " << k << " m: " << m << std::endl;
 
-  float At[n*k], Bt[k*m];
+  float *At = new float[n*k];
+  float *Bt = new float[k*m];
+
   transpose(A, At, n, k);
   transpose(B, Bt, k, m);
 
   gemm(dC, Bt, dA, n, k, m);
   gemm(At, dC, dB, n, k, m);
 
+  free(At);
```

Comment:
delete []At;
delete []Bt;

new and delete are matching, and malloc and free are matching.

Reply:
thanks for pointing it out!